python_ta/typecheck/type_store.py

```diff
@@ -64,39 +65,27 @@ def _parse_functions(self, module: astroid.Module) -> None:
     def lookup_function(self, operator, *args):
         """Helper method to lookup a function type given the operator and types of arguments."""
```

Comment:
I can fix the code duplication here, but this function isn't being used anywhere.